hedera-node/yahcli/src/test/java/com/hedera/services/yahcli/test/scenarios/AccountsCommandsTest.java

```diff
@@ -47,6 +54,36 @@ final Stream<DynamicTest> readmeAccountsCreateExample() {
                         .has(accountWith().balance(ONE_HBAR).memo("Who danced between"))));
     }
 
+    @HapiTest
+    final Stream<DynamicTest> reKeyAccount() {
+        final var testAccountId = new AtomicReference<Long>();
+        return hapiTest(
+                cryptoCreate("testAccount")
+                        .exposingCreatedIdTo(accountID -> testAccountId.set(accountID.getAccountNum())),
+                sourcing(() -> newKeyNamed("testKey")
+                        .shape(SigControl.ED25519_ON)
+                        .exportingTo(() -> asYcDefaultNetworkKey("account" + testAccountId + ".pem"), "keypass")),
+                cryptoUpdate("testAccount").key("testKey"),
+                // rekey using yahcli with a newly generated key
+                sourcingContextual(spec -> yahcliAccounts("rekey", "-g", String.valueOf(testAccountId))
+                        .exposingOutputTo(output -> assertTrue(output.contains(testAccountId + " has been re-keyed")))),
+                sourcing(() -> newKeyNamed("newKey")
+                        .shape(SigControl.ED25519_ON)
+                        .exportingTo(() -> asYcDefaultNetworkKey("newKey.pem"), "keypass")),
```

Comment:
Ahh I see, this statement is just preparation for the next explicit key file command. Let's either move this into a new test (we can use `Order(1)`), or put a comment above this line (and line 77) explaining why these ops are there. I was confused for a minute since the first rekey command should've output a key file too. ('Why does this recreate a new key file?' 😵‍💫 )